src/test/java/api/support/APITests.java

```diff
@@ -513,6 +520,17 @@ protected void loanHasLoanPolicyProperties(JsonObject loan,
       loanPolicy.getJson().getString("name"));
   }
 
+  protected void loanHasLoanPolicyProperties(JsonObject loan, JsonObject loanPolicy) {
+
+    hasProperty("loanPolicyId", loan, "loan", loanPolicy.getString("id"));
+    hasProperty("loanPolicy", loan, "loan");
+
+    JsonObject loanPolicyObject = loan.getJsonObject("loanPolicy");
+
+    hasProperty("name", loanPolicyObject, "loan policy",
+      loanPolicy.getString("name"));
+  }
```

Comment:
Why is this defined in the API tests base class, is it because there are similar shared methods already defined?

Reply:
Yes, similar loanHasOverdueFinePolicyProperties, loanHasLostItemPolicyProperties methods defined in the same class

Reply:
I likely should have commented on those during other reviews :-/ 

If possible, please could this be moved to a custom matcher that can be used directly in the tests.

Reply:
moved

Reply:
Thanks